### ISSUE TITLE
DSF won't accept EasyID3

### ISSUE THREAD

**Issue by ThomasZett** [quodlibet/mutagen#362]
I try to modify tags on DSF files using EasyID3, butthis is not supported.

Is this hard to add?

br, ThomasZ

**Comment by lazka** [comment]
Yeah, all formats embedding id3 tags currently don't support the easy variants.

Not sure how hard it would be.
